frontend/package.json

```diff
@@ -6,6 +6,7 @@
   "author": "Boris Cherny <boris@performancejs.com>",
   "license": "Apache-2.0",
   "scripts": {
+    "build_cache": "ts-node scripts/build_cache.ts",
```

Comment:
looks like the script key should be kebap case. should the file name also be camelCase?